clustering/infinispan/extension/src/main/java/org/jboss/as/clustering/infinispan/subsystem/CacheContainerResourceDefinition.java

```diff
@@ -195,12 +195,12 @@ static void buildTransformation(ModelVersion version, ResourceTransformationDesc
         if (InfinispanModel.VERSION_4_0_0.requiresTransformation(version)) {
             builder.discardChildResource(NoTransportResourceDefinition.PATH);
 
-            EnumSet.allOf(ThreadPoolResourceDefinition.class).forEach(pool -> builder.addChildResource(pool.getPathElement(), pool.getDiscardPolicy()));
+            EnumSet.complementOf(EnumSet.of(ThreadPoolResourceDefinition.CLIENT)).forEach(pool -> builder.addChildResource(pool.getPathElement(), pool.getDiscardPolicy()));
```

Comment:
Don't worry about translating these lambdas - they are already addressed by https://github.com/wildfly/wildfly/pull/10965.
You will have to rebase though.

Reply:
I might as well, should make rebasing easier.